lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -42,9 +42,19 @@ function subscribeEvents() {
   });
 }
 
+function exists() {
```

Comment:
Nope, it only needs to be exported if it's directly used in the other module. Otherwise the `sendEvent` function keeps the scope of this module when looking for `exists`.